test/aave/aave-tests.js

```diff
@@ -437,47 +482,39 @@ const aaveClaimStkAaveTest = async () => {
         this.timeout(150000);
 
         const stkAaveAddr = '0x4da27a545c0c5B758a6BA100e3a049001de870f5';
-
+        const OWNER_ACC = '0xe97f5363b77424332aae94e358a41ccbeb9e454b';
```

Comment:
Would change to `USER_ACC` `CLAIM_ACC` as we use OWNER_ACC already in utils.js with a diff. meaning

Reply:
Fixed